src/guacenc/ffmpeg-compat.c

```diff
@@ -51,8 +51,38 @@
  */
 static int guacenc_write_packet(guacenc_video* video, void* data, int size) {
 
-    /* Write data, logging any errors */
-    if (fwrite(data, 1, size, video->output) == 0) {
+    int ret;
+
+#if LIBAVCODEC_VERSION_INT < AV_VERSION_INT(54,1,0)
+    AVPacket pkt;
+    /* have to create a packet around the encoded data we have */
```

Comment:
This isn't a deal-breaker for me, but as a point of consistency in style, this comment (`/* all lowercase */`) doesn't match the style of existing comments (`/* Sentence case */`). I would recommend making things consistent.